tools/OpenConsole.psm1

```diff
@@ -430,4 +430,12 @@ function Invoke-CodeFormat() {
     }
 }
 
-Export-ModuleMember -Function Set-MsbuildDevEnvironment,Invoke-OpenConsoleTests,Invoke-OpenConsoleBuild,Start-OpenConsole,Debug-OpenConsole,Invoke-CodeFormat,Invoke-XamlFormat,Verify-XamlFormat
+#.SYNOPSIS
+# Download clang-format.exe required for code formatting
+function Get-Format()

```

Comment:
I'm not in love with the exposed member name here, but... this will be fine. `Get-` isn't totally correct for what this does; it's more of an `Install-`